README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
+
+Install Go Task
+
+```sh
+brew install go-task
+```
```

Comment:
Provide context or a note that 'task dev' bundles server running and asset building, as users may not have prior knowledge about what operations this command performs. This will improve readability and user experience.